blocks/breadcrumbs/breadcrumbs-create.js

```diff
@@ -33,9 +33,7 @@ function renderBreadcrumb(breadcrumbs) {
 }
 
 export default async function createBreadcrumbs(container) {
-  const breadCrumbsCSS = new Promise((resolve) => {
-    loadCSS('/blocks/breadcrumbs/breadcrumbs.css', (e) => resolve(e));
-  });
+  const cssLoaded = loadCSS('/blocks/breadcrumbs/breadcrumbs.css');
```

Comment:
why not `const cssLoaded = await loadCSS('/blocks/breadcrumbs/breadcrumbs.css');`?